invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/theme.less

```diff
@@ -9,6 +9,7 @@
 @import "header";
 @import "footer";
 @import "overrides";
+@import "accounts";
```

Comment:
This file has conflict due to the current changes in the master branch, Pull the changes from master first.